lib/streamlit/file_util.py

```diff
@@ -15,17 +15,49 @@
 
 import contextlib
 import errno
+import io
 import os
 
 import fnmatch
 
 from streamlit import env_util
 from streamlit import util
+from streamlit.string_util import is_binary_string
 
 # Configuration and credentials are stored inside the ~/.streamlit folder
 CONFIG_FOLDER_NAME = ".streamlit"
```

Comment:
Should this be in a common file?

Reply:
I don't follow - this utility function *is* in a common file. (Do you think it should't be?)